rpc/client.go

```diff
@@ -348,7 +348,7 @@ func (c *Client) CallContext(ctx context.Context, result interface{}, method str
 		if result == nil {
 			return nil
 		}
-		return json.Unmarshal(resp.Result, result)
+		return json.Unmarshal(resp.Result, &result)
```

Comment:
It says on line `318`:
```
// The result must be a pointer so that package json can unmarshal into it. You
// can also pass nil, in which case the result is ignored.
```
So this fix looks incorrect to me... (?)

Reply:
> It says on line `318`:
> 
> ```
> // The result must be a pointer so that package json can unmarshal into it. You
> // can also pass nil, in which case the result is ignored.
> ```
> 
> So this fix looks incorrect to me... (?)

<img width="453" alt="image" src="https://user-images.githubusercontent.com/22344498/220153085-be928e1e-deb2-4aaf-895b-59b11c8137b5.png">
It is built correct in my side, did you on this 6d0c35ebca0ce6b0fc1658ce1d8216075dd5af8d PR

Reply:
Correct!!!